packages/sync-server/README.md

```
@@ -20,6 +20,38 @@ npm install --location=global @actual-app/sync-server
 
 After installing, you can execute actual-server commands directly in your terminal.
 
+> **npm 11+: `allow-scripts` warnings during install**
+>
+> The install above prints warnings like:
+>
+> ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)